concordium-sdk/src/main/java/com/concordium/sdk/crypto/ed25519/ED25519PublicKey.java

```diff
@@ -2,36 +2,42 @@
 
 import com.concordium.sdk.exceptions.ED25519Exception;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
-import lombok.ToString;
 import org.apache.commons.codec.DecoderException;
 import org.apache.commons.codec.binary.Hex;
 
+import static java.util.Arrays.copyOf;
+
 @Getter
 @EqualsAndHashCode
-@ToString
 public final class ED25519PublicKey {
     private final byte[] bytes;
 
     private ED25519PublicKey(byte[] bytes) {
         this.bytes = bytes;
     }
 
+    public byte[] getBytes() {
+        return copyOf(bytes, bytes.length);
+    }
+
     @JsonCreator
-    public static ED25519PublicKey from(String hexKey) {
+    public static ED25519PublicKey from(String hexKey) throws ED25519Exception {
```

Comment:
There is no reason to declare the `ED25519Exception` here as it is **now** unchecked. (See my comment below)
Further it is not thrown by this function, but from the one below. 

I suggest removing the `throws` from this function, 
and declare it in the java doc of the function below. 

And also please add documentation to these public functions. What do they expect what to do they return, when do they throw `IllegalArgumentException` and `ED25519Exception` .

Reply:
done